### ISSUE TITLE
[Feature Request] Add Support for MultiPaper

### ISSUE THREAD

**Issue by DerLinkman** [Skoice/skoice#43]
Hello everyone!

I´ve just found this System here on the Proximity Voice market and really liked it in comparison to other plugins of that kind.

It´s extremely easy to setup and is very intuitive to use.

I´ve recently tried a experiment so to say about: "Can Skoice run on a multiserver MultiPaper Server?"

The result: Theoretically yes, but practical no.

The Issue: Skoice does not see the players in the near when a player is not on the same server (which is clear since it is originally written for Spigot/Paper and not Multipaper).

Long story short i would like to ask if it would be possible to implement a MultiPaper Support for this plugin?

Unfortunately i`m not a Java Developer so i cannot do it on my own (believe i would if i could) but i can provide you with useful help if needed in case of testing or general informations about the implementation idea.

**Comment by carlodrift** [comment]
Hello, thank you for your interest in Skoice. #27 will allow MultiPaper support. Thanks again for your suggestion and see you soon.

**Comment by DerLinkman** [comment]
Oh i´m sorry that i´ve missed that... but very very nice to hear! Thanks
